home-page

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Binabox NFT Collection</title>
+</head>
+<body>
+
+    <h2>Binabox</h2>
```

Comment:
It's really cool that the title is used here! It is very important for us to keep track of the order of the title. That is, if there is no h1 on the page, then h2 cannot be either. Another example, if we have h1, h2 and h4, and h3 is omitted, then this is also incorrect, h3 should be